### PR TITLE
feature/se-39

### LINKED ISSUE #39: Create the Front-end for the Dashboard
image

### PR DESCRIPTION
**Pull request by Chrissydrx**
image
close SE2-WS23/documentation#39

### CODE REVIEW

File: src/pages/Dashboard/DashboardPage.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      link: "#",
+    },
+    {
+      text: "Job Profiles",
+      link: "#",
+    },
+  ];
+
+  return (
+    <PageLayout>
```
Comment (julix14) [inline src/pages/Dashboard/DashboardPage.jsx]: Pass the Page Name pls

**Review by julix14 — CHANGES_REQUESTED** [review]
image
I would like to have there a proper title

File: src/pages/Dashboard/DashboardPage.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      <Box
+        sx={{
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "stretch",
+          //56px and 64px are the height of the toolbar specified in the
+          //theme for different viewports
+          height: { xs: "calc(100vh - 56px)", sm: "calc(100vh - 64px)" },
```
Comment (julix14) [inline src/pages/Dashboard/DashboardPage.jsx]: I think we need this on every page, is there a way to put that into a theme?

Comment (Chrissydrx) [reply src/pages/Dashboard/DashboardPage.jsx]: I think not every page need it, but I can do this as a prop. So you can opt in. Is this fine?

**Review by julix14 — APPROVED** [review]
image
This image was half an hour in my clipboard...but the wait was worth it

**Pull request merged**